idb/idb.go

```diff
@@ -252,6 +252,8 @@ type AccountQueryOptions struct {
 
 	IncludeAssetHoldings bool
 	IncludeAssetParams   bool
+	// IncludeMinBalance indicates whether to calculate min-balance as well (a realatively slow operation)
+	IncludeMinBalance bool
```

Comment:
These are database layer options, so this option doesn't need to be here. I think we can get away with using the nil round check, or just skip/unset it in the rewind function.